src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -47,6 +47,34 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
   ui_->unwindingMethodComboBox->addItem("DWARF", static_cast<int>(CaptureOptions::kDwarf));
   ui_->unwindingMethodComboBox->addItem("Frame pointers",
                                         static_cast<int>(CaptureOptions::kFramePointers));
+
+  const QString kFramePointerTooltip =
+      "Frame-pointer unwinding is experimental.\n"
+      "Successful unwinding requires all binaries to be compiled with\n"
+      "'-fno-omit-frame-pointer', but '-momit-leaf-frame-pointer' is\n"
+      "also supported.";
+
+  int frame_pointer_item_index =
+      ui_->unwindingMethodComboBox->findData(static_cast<int>(CaptureOptions::kFramePointers));
+  ui_->unwindingMethodComboBox->setItemData(frame_pointer_item_index, QColor(255, 128, 0, 255),
+                                            Qt::ForegroundRole);
+  ui_->unwindingMethodComboBox->setItemData(frame_pointer_item_index, kFramePointerTooltip,
+                                            Qt::ToolTipRole);
+
+  QObject::connect(
+      ui_->unwindingMethodComboBox, &QComboBox::currentTextChanged, ui_->unwindingMethodWidget,
+      [this, kFramePointerTooltip]() {
+        auto unwinding_method =
+            static_cast<UnwindingMethod>(ui_->unwindingMethodComboBox->currentData().toInt());
+        if (unwinding_method == CaptureOptions::kFramePointers) {
+          ui_->unwindingMethodComboBox->setStyleSheet("QComboBox:editable { color:#ff8000; }");
+          ui_->unwindingMethodWidget->setToolTip(kFramePointerTooltip);
+        } else {
+          ui_->unwindingMethodComboBox->setStyleSheet("QComboBox:editable { color : white; }");
+          ui_->unwindingMethodWidget->setToolTip("");
+        }
+      });
```

Comment:
Is that still needed after your recent change?

Reply:
That would be still needed. Otherwise tooltip and color for the widget/current item in the combobox would not work.